BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -98,6 +98,7 @@ public struct StandardImageIdentifiers {
         public static let pageZoomFill = "pageZoomFillLarge"
         public static let pin = "pinLarge"
         public static let pinSlash = "pinSlashLarge"
+        public static let pinSlashFilled = "pinSlashFillLarge"
```

Comment:
Nit: it should be `pinSlashFill` so the automated script updates it correctly later if needed